docs/api/backend-api/应用工具类管理.md

```
@@ -0,0 +1,160 @@
+### 基本信息
+
+**Path：** /app-center/api/apps/extension/update
+
+**Method：** POST
+
+**接口描述：**
+
+
+### 请求参数
+
+**Headers**
```

Markdownlint (0.37.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)